<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>logit</class>
 <widget class="QMainWindow" name="logit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>logit</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>891</width>
      <height>631</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>50</y>
        <width>301</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>提示信息：</string>
      </property>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>31</y>
         <width>271</width>
         <height>161</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>310</y>
        <width>251</width>
        <height>121</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pushButton_register2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>注册</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Button_back">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>225</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>ID：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_ID2"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>密码：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_password2"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>10</y>
        <width>911</width>
        <height>721</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/reses/res/backgroud/bac_cat2.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>201</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#aaaaff;&quot;&gt;用户注册&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <zorder>label_4</zorder>
     <zorder>groupBox</zorder>
     <zorder>frame</zorder>
     <zorder>label_10</zorder>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>70</y>
        <width>301</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>提示信息：</string>
      </property>
      <widget class="QTextBrowser" name="text_logitInfo">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>271</width>
         <height>161</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>350</y>
        <width>311</width>
        <height>121</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="Button_logit">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>80</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>登录</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelUserHead">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>61</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resources.qrc">:/reses/res/icons/icon_cat8.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="Button_register">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>80</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>注册</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_ID">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>12</y>
         <width>171</width>
         <height>21</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>用户名</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>78</x>
         <y>40</y>
         <width>45</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>密码：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>78</x>
         <y>12</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ID：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_password">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>40</y>
         <width>171</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>100</y>
        <width>311</width>
        <height>211</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="Button_connect">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>120</y>
         <width>121</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>连接</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>11</y>
         <width>285</width>
         <height>79</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>IP地址：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_IP"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>信息传输端口：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_info"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>文件传输端口：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_file"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>301</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>提示信息：</string>
      </property>
      <widget class="QTextBrowser" name="text_logitInfo_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>271</width>
         <height>161</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>959</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
